app/src/main/java/in/testpress/testpress/ui/ItemListFragment.java

```diff
@@ -166,6 +172,9 @@ public boolean onOptionsItemSelected(final MenuItem item) {
             return false;
         }
         switch (item.getItemId()) {
+            case id.products:
```

Comment:
This is wrong. ItemListFragment is an independent abstract class. Concrete cases like these should be implemented in the class which inherits this.